lerna.json

```diff
@@ -0,0 +1,13 @@
+{
+  "version": "5.0.0",
+  "command": {
+    "publish": {
+      "registry": "https://registry.npmjs.org"
+    }
+  },
+  "packages": [
+    "packages/*"
+  ],
+  "npmClient": "yarn",
+  "useWorkspaces": true
```

Comment:
There's a pretty serious bug in Yarn v1 relating to workspaces: the `--frozen-lockfile` flag doesn't work properly. See here for details: https://github.com/yarnpkg/yarn/issues/4098

If we're going to use Yarn workspaces, we should update the `deps-install` script to omit the `--frozen-lockfile` flag and check manually whether there have been any lockfile changes (e.g. check to ensure the working tree is still unchanged).

Reply:
0214dc0